polyfills/HTMLCanvasElement/protoype/toBlob/config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"browsers": {
+		"chrome": "<50",
```

Comment:
Would it be possible to add a version for all our browsers? https://polyfill.io/v2/docs/contributing/authoring-polyfills#browsers